tests/rspec_webui_tests.pm

```diff
@@ -4,8 +4,8 @@ use testapi;
 
 sub run() {
         my $branch = "master";
-        if ($VERSION != "Unstable") {
-          $branch = $VERSION;
+        if (get_var('VERSION') != "Unstable") {
```

Comment:
let me tell you a little detail about perl: != only compares numbers :)

But you want 
```
if (!check_var('VERSION', 'Unstable')) ...
```